src/ir/types/type_test.cc

```diff
@@ -303,4 +303,10 @@ INSTANTIATE_TEST_SUITE_P(
     RoundTripTypeProtoThroughTypeTest,
     testing::ValuesIn(type_proto_and_access_path_strings));
 
+// TODO(#122): This test should be moved to appropriate file while refactoring.
+TEST(EntityTypeTest, KindReturnsCorrectKind) {
+  EntityType entity_type(Schema(std::nullopt, {}));
```

Comment:
This is using `std::nullopt` when the `Schema` signature uses `absl::optional`. I'm guessing that this works because `absl::optional` is actually just `typedef`ed to `std::optional` internally for C++17. However, we should probably use `absl::nullopt` for consistency reasons.